src/SDKs/Automation/Management.Automation/generate.ps1

```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "automation/resource-manager" -PowershellInvoker  -AutoRestVersion "latest"
```

Comment:
Please regenerate the code using this script.

Reply:
The `src/SDKs/_metadata/automation_resource-manager.txt` log file tells us when code has been generated using powershell. We're slowly moving most RPs towards using ps1 scripts instead of cmds in order to provide better usage